java/src/jmri/jmrit/beantable/LRouteTableBundle_ca.properties

```diff
@@ -1,6 +1,6 @@
-# LRouteTableBundle.properties
+# L'ItinerariTableBundle.properties
 #
-# Default properties for the jmri.jmrit.beantable.LRoute GUI elements
+# Default properties for the jmri.jmrit.beantable.L'Itinerari GUI elements
```

Comment:
You shouldn't translate every place where you find the word LRoute, like the file name (line 1 and this one).